scripts/appveyor-tool.ps1

```diff
@@ -129,6 +129,7 @@ Function InstallRtools {
     $gcc_path = $env:GCC_PATH
   }
   $env:PATH = $RtoolsDrive + '\Rtools\bin;' + $RtoolsDrive + '\Rtools\MinGW\bin;' + $RtoolsDrive + '\Rtools\' + $gcc_path + '\bin;' + $env:PATH
+  $env:BINPREF=$RtoolsDrive + '/Rtools/mingw_$(WIN)/bin/'
```

Comment:
Is this really necessary if PATH contains only the path to the appropriate bin directory (as selected by $gcc_path in the line above)?